src/mono/System.Private.CoreLib/src/System/TypeLoadException.Mono.cs

```diff
@@ -24,7 +24,7 @@ private void SetMessageField()
                 return;
             }
 
-            _message = SR.Format("Could not load type '{0}' from assembly '{1}'.", _className, _assemblyName ?? SR.IO_UnknownFileName);
+            _message = string.Format(SR.ClassLoad_General, _className, _assemblyName ?? SR.IO_UnknownFileName);
```

Comment:
```suggestion
            _message = SR.Format(SR.ClassLoad_General, _className, _assemblyName ?? SR.IO_UnknownFileName);
```

Reply:
sorry what do you mean by this?
think i already added that in